### DISCUSSION TITLE
DurationField missing in SimpleMetadata.label_lookup

### DISCUSSION THREAD

**Discussion by smt-insens** [encode/django-rest-framework#8696] [Potential Issue] [answered]
### Context: 

I made an HTTP OPTIONS request. In the response I have the type of each field. I have a duration field so I expected to see something like "duration" as type but instead I have "field".

### Solution:

After investigation I found that `DurationField` is missing in `SimpleMetadata.label_lookup`

This is probably due to the fact `DurationField` was added to Django after the other fields. The person who added `DurationField` probably forgot to add it to the label_lookup.

The following line should be added to `label_lookup`: `serializers.DurationField: 'duration',`

https://github.com/encode/django-rest-framework/blob/20d347a806baa0dd481b31a7f847386574882d5b/rest_framework/metadata.py#L36-L58

https://github.com/encode/django-rest-framework/blob/20d347a806baa0dd481b31a7f847386574882d5b/rest_framework/fields.py#L1320-L1349

**Comment by lovelydinosaur** [comment, marked as answer]
Seems like you've got this one sorted.
Thanks for the detailed description.
Welcome to make a pull request.

**Reply by smt-insens** [reply]
@tomchristie here it is: https://github.com/encode/django-rest-framework/pull/8702
